docs/upgrade_1-2_to_1-3.md

```diff
@@ -42,15 +42,15 @@
 
 ## Upgrade steps
 
-### If the OADP 1.2 tech-preview Data Mover feature is in use, please read the following.
+### If the OADP 1.2 tech-preview Data Mover feature is in use, please read the following
 
 OADP 1.2 Data Mover backups can **NOT** be restored with OADP 1.3. To prevent a gap in the data protection of your applications we recommend the following to be **completed prior to the OADP upgrade**.
 
 * If on cluster backups are sufficient and CSI storage is available
   * Backup the applications with a CSI backup
 
 * If off cluster backups are required
-  * Backup the applications with a filesystem backup using the `--default-volumes-to-fs-backup=true` option.
+  * Backup the applications with a filesystem backup using the `--default-volumes-to-fs-backup=true` or `backup.spec.defaultVolumesToFsBackup` options.
   * Backup the applications with your CloudStorage plugins e.g. velero-plugin-for-aws
```

Comment:
I would rename this object storage plugins for clarity and consistency with velero lingo.

In addition, we have CloudStorage API that can be confused with this wording.

Reply:
good point